<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>resultProc</class>
 <widget class="QWidget" name="resultProc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>297</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>ID:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="ID_Lb">
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>ID:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Tiempo Máximo:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="TMax_Lb">
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TM:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Operación:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="OP_Lb">
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>OP:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Resultado:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="Res_Lb">
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Res:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Tiempo Llegada:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="TLleg_Lb">
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TLleg:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Tiempo Finalización:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="TFin_Lb">
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TFin:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Tiempo Retorno: </string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="TRet_Lb">
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TRet:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_8">
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Tiempo Respuesta:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLabel" name="TRes_Lb">
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TResp:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_9">
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Tiempo Espera:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLabel" name="TEsp_Lb">
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TEspera:</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_10">
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Tiempo Servicio: </string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QLabel" name="TServ_Lb">
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TServ:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
